packages/4-xdapp-layer/solana/src/utils/wormhole/coder/idl.ts

```diff
@@ -54,12 +54,6 @@ export class IdlCoder {
       case 'i128': {
         return borsh.i128(fieldName);
       }
-      case 'u256': {
-        return borsh.u256(fieldName);
-      }
-      case 'i256': {
-        return borsh.i256(fieldName);
-      }
```

Comment:
these are not defined and typescript cries

Reply:
Can you check what version of this package is in the old SDK? I hesitate to just delete without knowing what this code does

Reply:
https://github.com/wormhole-foundation/wormhole/blob/main/sdk/js/src/solana/anchor/idl.ts#L113 defined here, apparently lifted from https://github.com/coral-xyz/anchor/blob/master/ts/packages/anchor/src/idl.ts#L111